app/src/mixins/common-mixin.ts

```diff
@@ -240,6 +236,18 @@ export class CommonMixin extends Vue {
     ].includes(type)
   }
 
+  /** Set store value of isLearBusiness flag by fetching business from Lear. */
+  async checkBusinessInLear (identifier: string): Promise<void> {
+    if (identifier) {
+      const fetchedBusiness = await BusinessServices.fetchBusiness(identifier)
+      if (fetchedBusiness) {
+        this.setIsLearBusiness(true)
+      } else {
+        this.setIsLearBusiness(false)
+      }
+    }
+  }
```

Comment:
For the record (no need to change this), another way to do this is to have it as an async store action.